crates/bevy_core_pipeline/src/bloom/mod.rs

```diff
@@ -637,8 +578,8 @@ fn prepare_bloom_textures(
         }) = camera.physical_viewport_size
         {
             let min_view = width.min(height) as f32;
-            // How many times we can halve the resolution, minus 3 to avoid tiny mips
-            let mip_count = (min_view.log2().round() as i32 - 3).max(1) as u32;
+            // How many times we can halve the resolution
+            let mip_count = (min_view.log2().floor() as u32).max(1);
```

Comment:
I'm still not convinced we want to go down to 1x1 textures - that's super small and seems unnecessary. 4x4 should be the limit imo. We can leave it if you think it's worthwhile, though.

Reply:
Using really low res highest level mips introduced visual artifacts in the original implementation.

Reply:
I believe I tested the new version on bistro, and it was fine with 1x1. We should double check again, though.

Reply:
I added a slight limit but not going to 1x1 means out maximum scattering angel is not 90deg. Oh well whatever, it wasn't really 1x1 anyway unless you had a perfectly square window